pkg/store/bucket.go

```diff
@@ -1137,6 +1146,7 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 		s.metrics.cachedPostingsCompressionTimeSeconds.WithLabelValues(labelDecode).Add(stats.CachedPostingsDecompressionTimeSum.Seconds())
 		s.metrics.cachedPostingsOriginalSizeBytes.Add(float64(stats.CachedPostingsOriginalSizeSum))
 		s.metrics.cachedPostingsCompressedSizeBytes.Add(float64(stats.CachedPostingsCompressedSizeSum))
+		s.metrics.postingsSizeBytes.Observe(float64(int(stats.PostingsFetchedSizeSum) + int(stats.PostingsTouchedSizeSum)))
```

Comment:
@GiedriusS: Am I correct in assuming that "touched" means downloaded from cache and "fetched" means downloaded from object storage? Seems like so but I am not sure. 🤔

Reply:
yep :+1:

Reply:
One thing to note is that, we also update `PostingsTouchedSizeSum` when we write encoded postings back to cache even if we fetch those postings from object storage. I feel like this behavior makes the metric not that accurate.

Reply:
Yeah I don't know if this is by design. @GiedriusS Do you know more about this